### ISSUE TITLE
Prophetic discharge and prophetic finesse should never be treated as unnecessary

### ISSUE THREAD

**Issue by Lel0uch-H** [hanabi/hanabi.github.io#821]
Both of these conventions get at least 2 cards for 1 clue already, I don't think mandating it to consider it unnecessary is good for the game.

@pianoblook I am curious why the convention is written the way it is currently.

**Comment by pianoblook** [comment]
The current wording is that prophetic *Finesses* are always unnecessary; indeed just checked the Proph.Discharge, which says:
> Unlike the Prophetic Finesse, the clued player should only treat this as unnecessary move if the Discharged card could have been cleanly clued.

The main reason why is that they're both actually just 1-for-1s. Like, literally the only reason Bob blind-plays is *because* the other copy is seen on finesse, and is therefore very likely going to be used as a 2-for-1 already. So IMO they fall perfectly into the category of _Unnecessary Use of Complicated Moves_.

Also, if memory serves, therefore adding the Unnecessary component was one of the key parts that got this to pass in the first place.

**Comment by Lel0uch-H** [comment]
> they're both actually just 1-for-1s.

Well... they are not... for example, when Alice clues Cathy, Bob responds and Donald's card(s) play from finesse position. So they are at least a 2-for-1.

**Comment by pianoblook** [comment]
right sorry, I didn't explain that well at all. I mixed up what I was trying to say- lemme try again. What I meant to express is that I *treat* them more as a 1-for-1s.

So yes, a Proph. Finesse will result in 2 plays total. *but* the actual added value of 'getting' those two cards is extremely low - and, I'd argue, often even negative. By forcing the prophetic you've lost *two* opportunities for other bluffs/finesses. 

In other words, since by definition the other copy is on _Finesse_, it feels like a very artificial 2-for-1.

**Comment by Zamiell** [comment]
both of these moves narrowly escaped deletion in this round of convention deletions - who knows if they will survive the next round.
either way you configure it, it is going to be a pretty minor difference

i've never seen either move so I don't have much to contribute, other than argreeing with Lelouch that it is more intuitive for the group to treat them as necessary since they are 2-for-1s
